Formula/jasper.rb

```diff
@@ -15,23 +15,34 @@ class Jasper < Formula
   depends_on "cmake" => :build
   depends_on "jpeg"
 
+  on_linux do
+    depends_on "freeglut"
+  end
+
   def install
     mkdir "build" do
-      # Make sure macOS's GLUT.framework is used, not XQuartz or freeglut
-      # Reported to CMake upstream 4 Apr 2016 https://gitlab.kitware.com/cmake/cmake/issues/16045
-      glut_lib = "#{MacOS.sdk_path}/System/Library/Frameworks/GLUT.framework"
+      os_cmake_args = []
```

Comment:
```suggestion
      args = std_cmake_args
      args << "-DJAS_ENABLE_DOC=OFF"
```

We can then remove usage of `std_cmake_args` below and simplify the formula a bit.